### ISSUE TITLE
resolveOnly option prevent local file resolution

### ISSUE THREAD

**Issue by arantes555** [rollup/plugins#209]
- Rollup Plugin Name: node-resolve
- Rollup Plugin Version: 7.1.0 & 7.1.1
- Rollup Version: 1.31.0
- Operating System (or Browser): macOS
- Node Version: v13.7.0

### How Do We Reproduce?

```bash
git clone https://github.com/arantes555/node-resolve-local-files.git
cd node-resolve-local-files
npm install
node build.js
```

### Expected Behavior

Using node-resolve 7.1.x with the `resolveOnly` option should prevent resolving other node modules, but should not prevent resolving local files. 

### Actual Behavior

Using node-resolve 7.1.x with the `resolveOnly` option prevents resolving local files.

**Comment by shellscape** [comment]
Thanks for reporting the issue. We'd be happy to review a PR that resolves this.

**Comment by shellscape** [comment]
Closed by #337
